druid.go

```diff
@@ -185,6 +188,14 @@ func WithBasicAuth(username, password string) ClientOption {
 	}
 }
 
+func WithSkipTLSVerify() ClientOption {
+	return func(opts *clientOptions) {
+		opts.httpClient.Transport = &http.Transport{
```

Comment:
eventually the `Transport` could already be set and then we would need to only set `TLSClientConfig` (here we override the whole transport).

Reply:
I did try to do this `opts.httpClient.Transport.(*http.Transport).TLSClientConfig.InsecureSkipVerify = true` it compiled but I got a crash at the same line 193 with this change `panic: interface conversion: http.RoundTripper is nil, not *http.Transport`

Reply:
Thanks for helping out in [slack](https://grafadruid.slack.com/archives/C01E6RA0TDM/p1614958861068300?thread_ts=1614786294.061900&cid=C01E6RA0TDM) with this.
Done!